pkgs/development/python-modules/pallets-sphinx-themes/default.nix

```
@@ -0,0 +1,24 @@
+{ lib, buildPythonPackage, fetchFromGitHub, sphinx, packaging }:
+
+buildPythonPackage rec {
+  pname = "pallets-sphinx-themes";
+  version = "2.0.2";
+
+  src = fetchFromGitHub {
+    owner = "pallets";
+    repo = "pallets-sphinx-themes";
+    rev = version;
+    sha256 = "0nvznv6abmkkda2fahydd4rykd94rmz74hx5aypv6j22zvf5pj8b";
+  };
+
+  propagatedBuildInputs = [ packaging sphinx ];
+
+  pythonImportsCheck = [ "pallets_sphinx_themes" ];
```

```suggestion
  pythonImportsCheck = [ "pallets_sphinx_themes" ];
  
  doCheck = false;
```

There are no tests, so we should disable the check phase.

---

Ok. Applied.